hooks/useUpdatePoolInstances/index.ts

```diff
@@ -21,13 +21,16 @@ import { fetchPendingCommits, V2_SUPPORTED_NETWORKS } from '~/utils/tracerAPI';
  * Wrapper to update all pools information
  */
 export const useUpdatePoolInstances = (): void => {
-    const { provider, account } = useStore(selectWeb3Info);
-    const poolAddresses = useStore(selectAllPoolLists);
-    const pools = useStore(selectPoolInstances, shallow);
+    const { setMultiplePools, resetPools, setPoolsInitialized, setTokenBalances } = useStore(
+        selectPoolInstanceActions,
+        shallow,
+    );
+    const { updateTokenApprovals, updateTokenBalances } = useStore(selectPoolInstanceUpdateActions, shallow);
+    const { addCommit } = useStore(selectUserCommitActions, shallow);
+    const { provider, account } = useStore(selectWeb3Info, shallow);
+    const poolAddresses = useStore(selectAllPoolLists, (oldState, newState) => oldState.length === newState.length);
```

Comment:
unlikely to happen but if switching between networks and both happen to have the same number of pools does will this get handled somewhere?

Reply:
mmmmm, ill change it back, this check ended up not doing anything anyway